extensions/amp-story-interactive/0.1/amp-story-interactive-img-quiz.css

```diff
@@ -60,11 +47,16 @@
 
 .i-amphtml-story-interactive-post-selection
   .i-amphtml-story-interactive-img-quiz-answer-choice::before {
+  content: '' !important;
+  display: block !important;
+  width: 100% !important;
+  height: 100% !important;
+  border-radius: 50% !important;
+  position: absolute !important;
```

Comment:
Nit: Do we need this change? I'd think it's better the other way (and also, it's not related to the ABCD coloring)